function/pkg/resources/unstructured/subscription.go

```diff
@@ -0,0 +1,126 @@
+package unstructured
+
+import (
+	"fmt"
+
+	"github.com/kyma-incubator/hydroform/function/pkg/resources/types"
+	"github.com/kyma-incubator/hydroform/function/pkg/workspace"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+const (
+	apiVersionSubscription = "eventing.kyma-project.io/v1alpha1"
+)
```

Comment:
```suggestion
const apiVersionSubscription = "eventing.kyma-project.io/v1alpha1"
```

According with function.go and git_repository.go